src/epub/text/scanners-live-in-vain.xhtml

```diff
@@ -464,7 +464,7 @@
 				<p>“Scanners?”</p>
 				<p>“Yes, darling? What is it?”</p>
 				<p>“Scanners?”</p>
-				<p>“Scanners. Oh, yes, darling, they’re all right. They had to arrest some of them for going into <em>High Speed</em> and running away. But the Instrumentality caught them all⁠—all those on the ground⁠—and they’re happy now. Do you know, darling,” she laughed, “some of them didn’t want to be restored to normality. But Stone and his Chiefs persuaded them.”</p>
+				<p>“Scanners. Oh, yes, darling, they’re all right. They had to arrest some of them for going into <em>High speed</em> and running away. But the Instrumentality caught them all⁠—all those on the ground⁠—and they’re happy now. Do you know, darling,” she laughed, “some of them didn’t want to be restored to normality. But Stone and his Chiefs persuaded them.”</p>
```

Comment:
I think this should remain titlecased

Reply:
Note that it's _not_ titlecased on lines 440 and 441. To be clear, you'd like it to remain titlecased here but not there?

Reply:
It should be titlecased everywhere

Reply:
Done.